app/views/base.py

```diff
@@ -37,6 +37,93 @@ class BaseView(django.views.View):
         'lang': utils.strip,
     }
 
+    class Env(object):
+        """Class to store environment information used by views and templates.
```

Comment:
Can you also document that a subclass of BaseView can define a nested `Env` class to be used as a replacement of this class? Since it is a bit unusual API.